[Middleware_02] Building TrackDataCalculatorClass

kwillard34 opened the pull request:
working on #64

kwillard34 committed:
Create TrackDataCalculatorTests.java

kwillard34 committed:
Update TrackDataCalculatorTests.java

kwillard34 committed:
Update TrackDataCalculatorTests.java

kwillard34 committed:
Update TrackDataCalculatorTests.java

tomtomtomcat committed:
Adjust import order (style guide)

tomtomtomcat committed:
adjust import order part 2 and whitespace adjustments

tomtomtomcat committed:
more style guide fixes

tomtomtomcat committed:
more style fixes

tomtomtomcat committed:
replace wildcard import with time.Duration

tomtomtomcat committed:
Merge branch 'main' into Building-TrackDataCalculatorClass

mckenzieking reviewed (approved):
looks good

kwillard34 committed:
production code

kwillard34 committed:
Update TrackDataCalculator.java

kwillard34 committed:
Update TrackDataCalculator.java

kwillard34 committed:
Update TrackDataCalculator.java

andrewdimmer committed:
[Middleware_02] Fixed Test Coverage of TrackDataCalculator

andrewdimmer committed:
[Middleware_02] Fix Checkstyle Issues in TrackDataCalculatorTests

Also cleaned some stuff up for code coverage.

andrewdimmer committed:
[Middleware_02] Adjust Speed Rounding

andrewdimmer reviewed (approved):
Looks good enough to me!

kwillard34 committed:
Merge branch 'main' into Building-TrackDataCalculatorClass

andrewdimmer committed:
[Middleware_02] Removed Round on the Speed Calculation

For some reason it was causing a large degree of error off of the expected result.

andrewdimmer committed:
[Middleware_02] Fixed test initialization to prevent roll-overs

andrewdimmer committed:
[Middleware_02] Fixed Speed int data type bug

If the time was less than an hour, the duration was 0, and the speed infinity...
